package.json

```diff
@@ -50,6 +50,7 @@
     "check-types": "^7.2.0",
     "fs": "0.0.1-security",
     "jimp": "^0.2.28",
+    "pkg-dir": "2.0.0",
```

Comment:
differencify version update not needed ?

Reply:
It does. Richard probably going to update this at some point

Reply:
I'm not entirely what you mean @nimishseraphim, but this branch is now up to date with master.

Reply:
I've updated this 👍